draft-ietf-quic-transport.md

```diff
@@ -2919,29 +2919,58 @@ CONNECTION_CLOSE frames is used to carry other frame-specific flags. For all
-| 0x02 - 0x03 | ACK                  | {{frame-ack}}                  |
-| 0x04        | RESET_STREAM         | {{frame-reset-stream}}         |
-| 0x05        | STOP_SENDING         | {{frame-stop-sending}}         |
-| 0x06        | CRYPTO               | {{frame-crypto}}               |
-| 0x07        | NEW_TOKEN            | {{frame-new-token}}            |
-| 0x08 - 0x0f | STREAM               | {{frame-stream}}               |
-| 0x10        | MAX_DATA             | {{frame-max-data}}             |
-| 0x11        | MAX_STREAM_DATA      | {{frame-max-stream-data}}      |
-| 0x12 - 0x13 | MAX_STREAMS          | {{frame-max-streams}}          |
-| 0x14        | DATA_BLOCKED         | {{frame-data-blocked}}         |
-| 0x15        | STREAM_DATA_BLOCKED  | {{frame-stream-data-blocked}}  |
-| 0x16 - 0x17 | STREAMS_BLOCKED      | {{frame-streams-blocked}}      |
-| 0x18        | NEW_CONNECTION_ID    | {{frame-new-connection-id}}    |
-| 0x19        | RETIRE_CONNECTION_ID | {{frame-retire-connection-id}} |
-| 0x1a        | PATH_CHALLENGE       | {{frame-path-challenge}}       |
-| 0x1b        | PATH_RESPONSE        | {{frame-path-response}}        |
-| 0x1c - 0x1d | CONNECTION_CLOSE     | {{frame-connection-close}}     |
+| Type Value  | Frame Type Name      | Definition                     | Packets |
+|:------------|:---------------------|:-------------------------------|---------|
+| 0x00        | PADDING              | {{frame-padding}}              | IH01    |
+| 0x01        | PING                 | {{frame-ping}}                 | IH01    |
+| 0x02 - 0x03 | ACK                  | {{frame-ack}}                  | IH_1    |
+| 0x04        | RESET_STREAM         | {{frame-reset-stream}}         | __01    |
+| 0x05        | STOP_SENDING         | {{frame-stop-sending}}         | __01    |
+| 0x06        | CRYPTO               | {{frame-crypto}}               | IH_1    |
+| 0x07        | NEW_TOKEN            | {{frame-new-token}}            | ___1    |
+| 0x08 - 0x0f | STREAM               | {{frame-stream}}               | __01    |
+| 0x10        | MAX_DATA             | {{frame-max-data}}             | __01    |
+| 0x11        | MAX_STREAM_DATA      | {{frame-max-stream-data}}      | __01    |
+| 0x12 - 0x13 | MAX_STREAMS          | {{frame-max-streams}}          | __01    |
+| 0x14        | DATA_BLOCKED         | {{frame-data-blocked}}         | __01    |
+| 0x15        | STREAM_DATA_BLOCKED  | {{frame-stream-data-blocked}}  | __01    |
+| 0x16 - 0x17 | STREAMS_BLOCKED      | {{frame-streams-blocked}}      | __01    |
+| 0x18        | NEW_CONNECTION_ID    | {{frame-new-connection-id}}    | __01    |
+| 0x19        | RETIRE_CONNECTION_ID | {{frame-retire-connection-id}} | __01    |
+| 0x1a        | PATH_CHALLENGE       | {{frame-path-challenge}}       | __01    |
+| 0x1b        | PATH_RESPONSE        | {{frame-path-response}}        | __01    |
+| 0x1c - 0x1d | CONNECTION_CLOSE     | {{frame-connection-close}}     | IH_1*   |
```

Comment:
We'll need to add HANDHAKE_DONE here once #3145 is merged.

Reply:
I'll send you another PR.  I'm merging this now, but I can help out that much.